### DISCUSSION TITLE
[Q&A] 使用PandoraNext api无法使用爬虫插件

### DISCUSSION THREAD

**Discussion by sfdzkj** [lobehub/lobehub#922] [General Question] [answered]
### 🧐 问题描述 | Proposed Solution

官网api 额度有限，使用PandoraNext API 调用官网网页实现免费。但 这种方式无法使用插件(爬虫无法使用)功能。 请问有什么解决办法吗？或者能否支持下 这种第三方API 使用插件

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
你用的是哪个 api ？如果是 hack 的 api 类型估计没法支持

[a comment by an automated account is not shown]

**Comment by fxecc** [comment]
@arvinxx  不明白，hack 是什么？我用的https://github.com/pandora-next/deploy，这个项目是代理官网的网页编程API。 我是多个账户合并的 pool_token

[a comment by an automated account is not shown]

**Comment by Feiyuyu0503** [comment, marked as answer]
@fxecc pandora-next是利用chatgpt web上的对话模拟api调用，不能像真的openai api支持函数调用使用插件
![image](https://github.com/lobehub/lobe-chat/assets/32660959/ad1a95eb-af53-4bef-a1dc-04b3f1eb0add)

**Reply by fxecc** [reply]
明白了，非常感谢1

[a comment by an automated account is not shown]
